extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -109,7 +110,10 @@ void FlowFileRepository::run() {
 }
 
 void FlowFileRepository::prune_stored_flowfiles() {
-  rocksdb::DB* stored_database_;
+  rocksdb::DB* stored_database_ = nullptr;
```

Comment:
This is an unrelated memory leak but I didn't want to create a separate PR for this.